source/_components/device_tracker.netgear.markdown

```diff
@@ -26,16 +26,41 @@ device_tracker:
     password: YOUR_ADMIN_PASSWORD
 ```
 
-Configuration variables:
-
-- **url** (*Optional*): The base URL, e.g., `http://routerlogin.com:5000` for example. If not provided `host` and `port` are used. If none provided autodetection of the URL will be used.
-- **host** (*Optional*): The IP address of your router, e.g., `192.168.1.1`.
-- **port** (*Optional*): The port your router communicates with.
-- **username** (*Optional*): The username of a user with administrative privileges. If not provided `admin` will be used.
-- **password** (*Required*): The password for your given admin account.
-- **devices** (*Optional*): If provided only specified devices will be reported. Can be MAC address or the device name as reported in the Netgear UI.
-- **exclude** (*Optional*): Devices to exclude from the scan.
-- **accesspoints** (*Optional*): Also track devices on the specified APs. Only supports MAC address.
+{% configuration %}
+url:
+  description: The base URL, e.g., `http://routerlogin.com:5000` for example. If not provided `host` and `port` are used. If none provided autodetection of the URL will be used.
+  required: false
+  type: string
+host:
+  description: The IP address of your router, e.g., `192.168.1.1`.
+  required: false
+  type: string
+port:
```

Comment:
Not really clear by the older docs and code, but the default here is actually `5000`.
Please add the `default` key.